docker-sync.yml

```diff
@@ -22,9 +22,9 @@ syncs:
     src: '.'
     sync_excludes: ['node_modules', 'logs']
   
-  rems_dev_pharmacy-information-system-sync:  
```

Comment:
Should this be removed or is it kept around for a reason?

Reply:
should be removed, good catch